app/models/user.rb

```diff
@@ -151,9 +151,9 @@ def mhv_correlation_id
     identity.mhv_correlation_id || mpi_mhv_correlation_id
   end
 
-  def mhv_user_account
+  def mhv_user_account(break_cache: false)
```

Comment:
This won't break the cache if `@mhv_user_account` exists, right?

Reply:
yeah, good call